doc/CONFIGURATION.md

```diff
@@ -235,189 +235,177 @@ Besides the `modules` and `config`, libp2p allows other internal options and con
 //   dht: kad-dht
 //   pubsub: gossipsub
 
-const Libp2p = require('libp2p')
-const TCP = require('libp2p-tcp')
-const WS = require('libp2p-websockets')
-const MPLEX = require('libp2p-mplex')
-const { NOISE } = require('libp2p-noise')
-const MulticastDNS = require('libp2p-mdns')
-const DHT = require('libp2p-kad-dht')
-const GossipSub = require('libp2p-gossipsub')
-
-const node = await Libp2p.create({
-  modules: {
-    transport: [
-      TCP,
-      new WS() // It can take instances too!
-    ],
-    streamMuxer: [MPLEX],
-    connEncryption: [NOISE],
-    peerDiscovery: [MulticastDNS],
-    dht: DHT,
-    pubsub: GossipSub
-  }
+import { createLibp2p } from 'libp2p'
+import { TCP } from '@libp2p/tcp'
+import { WebSockets } from '@libp2p/websockets'
+import { Mplex } from '@libp2p/mplex'
+import { Noise } from '@chainsafe/libp2p-noise'
+import { MulticastDNS } from '@libp2p/mdns'
+import { KadDHT } from '@libp2p/kad-dht'
+import { GossipSub } from 'libp2p-gossipsub'
+
+const node = await createLibp2p({
+  transports: [
+    TCP,
```

Comment:
I think this doesn't work anymore, same below where classes are used instead of instances.